qa/rpc-tests/p2p-acceptblock.py

```diff
@@ -278,8 +278,15 @@ def run_test(self):
 
         # 7. Send the missing block for the third time (now it is requested)
         test_node.send_message(msg_block(blocks_h2f[0]))
-
         test_node.sync_with_ping()
+
+        # Wait for the reorg to complete. It can be slower on some systems.
+        while self.nodes[0].getblockcount() != 290:
+            time.sleep(1)
+            j = j + 1
+            if (j > 60):
```

Comment:
no it's not 100% accurate but for the purpose of what we're trying to do it's fine IMO.  It doesn't matter if it's exactly 60 seconds , it's just there so the test can at some point finish and proceed without hanging up.